kingfisher_scrapy/exceptions.py

```diff
@@ -12,3 +12,7 @@ class SpiderArgumentError(KingfisherScrapyError):
 
 class MissingRequiredFieldError(KingfisherScrapyError, KeyError):
     """Raised when an item is missing a required field"""
+
+
+class LinksSpiderError(KingfisherScrapyError):
+    """Raised when a next link is invalid in LinksSpider"""
```

Comment:
Let's use a more descriptive class name, since it's possible the LinksSpider will implement other exceptions in the future:

```suggestion
class MissingNextLinkError(KingfisherScrapyError):
    """Raised when a next link is not found on the first page of results"""
```